### ISSUE TITLE
How can I implement batch insert ?

### ISSUE THREAD

**Issue by arsis1** [blynkkk/clickhouse4j#48]
I am new to clickhouse and clickhouse4j.
And I can't see any helpful document about batch insert.

I think more example codes would be very helpful.
Thanks.

**Comment by doom369** [comment]
@arsis1 agree, this is something we have to work on. For now you can view the tests as usage reference. Typically driver follow regular JDBC syntax:

        try (Connection connection = dataSource.getConnection();
             PreparedStatement ps = connection.prepareStatement("insert into table (id) values (?)");
         for (var entry : entries) {
             ps.setInt(1, entry.id);
             ps.addBatch();
         }
         ps.executeBatch();
       }

**Comment by arsis1** [comment]
@doom369 Thanks, very helpful for me.

**Comment by doom369** [comment]
I'll reopen and will close when put some examples into the main page.
